### PR TITLE
fix: Theme Auto broken

### PR DESCRIPTION
**Pull request by mazzi**

### CODE REVIEW

File: server/static_src/js/theme.js [hunk 1]
```diff
@@ -13,40 +13,40 @@
...
-            document.querySelectorAll("[data-bs-theme]").forEach(element => {
-                element.setAttribute('data-bs-theme', theme)
-            })
-            if (theme === 'dark') {
-                setDark()
-            }
-            if (theme === 'light') {
-                setLight()
-            }
+        }
```
Comment (np5) [inline server/static_src/js/theme.js:18]: Why not just "else" ?

File: server/static_src/js/theme.js [hunk 2]
```diff
@@ -13,40 +13,40 @@
...
+        if ( (theme === 'light') || (theme === 'auto' && window.matchMedia('(prefers-color-scheme: light)').matches) ) {
+            setLight()
         }
     }
     const setDark = () => {
         document.querySelectorAll([".light", "[class*='-light']"]).forEach(element => {
-            element.className = element.className.replace("light","dark")
+            element.className = element.className.replace(" light"," dark")
```
Comment (np5) [inline server/static_src/js/theme.js:25]: Those tricks need to be documented as agreed.

File: server/static_src/js/theme.js [hunk 3]
```diff
@@ -13,40 +13,40 @@
...
         })
+        document.querySelectorAll("[data-bs-theme]").forEach(element => {
+            element.setAttribute('data-bs-theme', 'dark')
+            document.documentElement.setAttribute('data-bs-theme', 'dark')
+        })
     }
     const setLight = () => {
         document.querySelectorAll([".dark", "[class*='-dark']"]).forEach(element => {
-            element.className = element.className.replace("dark","light")
+            element.className = element.className.replace(" dark"," light")
```
Comment (np5) [inline server/static_src/js/theme.js:39]: same

**Review by np5 — CHANGES_REQUESTED** [review]

**Comment by mazzi** [comment]
Done with the PR comments (one).
Documentation will follow in another ticket.

**Pull request merged**